front/src/NotificationWindow.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import styled from "styled-components";
+import faTimes from "@fortawesome/fontawesome-free-solid/faTimes";
+import FontAwesomeIcon from "@fortawesome/react-fontawesome";
+
+const Window = styled.div`
+  background-color : ${props => props.backgroundColor}
+  border-style: solid;
+  border-size: 4px;
+  width: 90%;
+  position: relative;
+  margin-left:auto;
+  margin-right:auto;
+`;
+
+const NotificationHeader = styled.p`
+  text-align: center;
+  font-size: 5vmin;
+`;
+
+const NotificationText = styled.p`
+  padding-left: 10px;
+  padding-right: 10px;
+  text-align: center;
+  font-size: 8vmin;
+`;
+
+const NotificationCloseButton = styled.button`
+  border: none;
+  background-color: transparent;
+  position: absolute;
+  top: 0;
+  right: 0;
+  width: 10vmin;
+  height: 10vmin;
+  &:focus {
+    outline: 0;
+  }
+  font-size: 2vmin;
+`;
+
+/**
+ * Creates a notification window
+ * Author: S.A. Tanja
+ *
+ * Props:
+ * - backgroundColor: sets the background color of the notification window
+ * - notificationType: sets the title of the notification window
+ * - onClick: on close function should be passed
+ * - children: the message of the notification window
+ */
+const NotificationWindow = props => {
+  return (
+    <Window backgroundColor={props.backgroundColor}>
+      <NotificationHeader>{props.notificationType}</NotificationHeader>
+      <NotificationCloseButton
+        onClick={() => {
+          props.onClick();
+        }}
```

Comment:
`onClick={() => props.onClick()}`

Reply:
This was an ESLint correction. Still change it to a single line?

Reply:
I think that'd break the build.

Also, why not call this prop onDismiss? Clicking on the notification itself could possibly be used for something useful

Reply:
prettier does not format this line for me, even after committing.

Reply:
Is your version of ESLint up-to-date? I have 1.4.9 at the moment. @helaan do you have the single line suggestion, or the multi line?

Reply:
we have the same version of eslint. it is one of the dependencies of the project.

Reply:
`yarn eslint front/src/*.js` gives the following when I format it to a single line:

  /home/henk/.go/src/github.com/rvolosatovs/turtlitto/front/src/NotificationWindow.js
  56:19  error  Replace `·onClick={()·=>·{props.onDismiss();}}` with `⏎········onClick={()·=>·{⏎··········props.onDismiss();⏎········}}⏎······`  prettier/prettier

This change is made automatically when running `make js.fmt`.

Reply:
ofc, `onClick={()·=>·{props.onDismiss();}}` is wrong.
do `onClick={() => props.onDismiss()}`

Reply:
That's pretty damn subtle